components/pages/home/Discord.stories.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react'
+import Discord from './Discord'
+
+const defExport = {
+  title: 'Pages/Home/Discord',
+  component: Discord,
+}
+
+export default defExport
+const Template = (args) => <Discord />
+export const Discord = Template.bind()
```

Comment:
Kena eslint pas lagi build, coba diganti jadi yang lain om

```suggestion
export const DiscordElement = Template.bind()
```

Reply:
Oh iya om.. siap" 👍🏻